department-of-veterans-affairs/vets-website
# STEM: Remove 22-10203s from in progress applications that were purposefully exited  #13794

## Description

Create `ExitApplicationButton` component that if a user is logged in will remove the 10203 from in progress applications on click

https://github.com/department-of-veterans-affairs/va.gov-team/issues/13794

## Testing done
- Logged in as user
- Started a 10203 and put in disqualifying choices
- Observed "Saved application at..." at above Continue and Back buttons while on either page with "Exit Application" button
- Open "My VA" profile in a separate tab
- Observe there is an in progress 10203 listed
- Return to 10203 tab and click "Exit Application"
- Return to "My VA" profile tab and reload
- Observe 10203 is no longer listed as in progress

## Screenshots
This isn't visible without adding `target="_blank"` to the button but if a form is not found to be deleted the call fails with a 404

![image](https://user-images.githubusercontent.com/1094999/94706560-57a93880-0310-11eb-9c7d-118925dd8ef5.png)


## Acceptance criteria

## Definition of done
